### PR TITLE
update troubleshooting guide for recent installation errors fix.

### PR DESCRIPTION
**Pull request by ashmeenkaur**
### Description
update troubleshooting guide for recent installation errors fix.
Change in rich text format: https://screenshot.googleplex.com/5233YcL3pMnzqow.png

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
...Bad gateway error while installing/upgrading GCSFuse:<br/> <pre>Err: http://packages.cloud.google.com/apt gcsfuse-focal/main amd64 gcsfuse amd64 1.2.0<br/>502  Bad Gateway [IP: xxx.xxx.xx.xxx 80]</pre>                                                                                                                                                                                                                                                                                                                                                                                                             | This error is seen when the url used in /etc/apt/sources.list.d/gcsfuse.list file uses HTTP protocol instead of HTTPS protocol. Run the following commands to update /etc/apt/sources.list.d/gcsfuse.list file with the https:// url.<br/> <pre>$ sudo rm /etc/apt/sources.list.d/gcsfuse.list <br/>$ export GCSFUSE_REPO=gcsfuse-`lsb_release -c -s<br/>$ echo \"deb https://packages.cloud.google.com/apt $GCSFUSE_REPO main\" &#124; sudo tee /etc/apt/sources.list.d/gcsfuse.list                                                                                                                                  |
```
Comment (gargnitingoogle) [inline docs/troubleshooting.md:23]: Looks like the `<pre>` tag before `sudo rm` in this line doesn't have a matching `</pre>` tag. I'm not sure if it causes an issue in markdown though.

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
...Repository changed 'Origin' and 'Label' error while running `apt-get update` command:  <pre>E: Repository 'http://packages.cloud.google.com/apt gcsfuse-focal InRelease' changed its 'Origin' value from 'gcsfuse-jessie' to 'namespaces/gcs-fuse-prod/repositories/gcsfuse-focal'<br/>E: Repository 'http://packages.cloud.google.com/apt gcsfuse-focal InRelease' changed its 'Label' value from 'gcsfuse-jessie' to 'namespaces/gcs-fuse-prod/repositories/gcsfuse-focal'<br/>N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details. </pre> | Use one of the following commands to upgrade to latest GCSFuse version <pre>sudo apt-get update --allow-releaseinfo-change <br>OR<br>sudo apt update -y && sudo apt-get update                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```
Comment (gargnitingoogle) [inline docs/troubleshooting.md:24]: Same as above. ` <pre>sudo apt-get update` doesn't have a matching `</pre>` tag.

**Review by gargnitingoogle — CHANGES_REQUESTED** [review]
@ashmeenkaur  Is it possible to take a screenshot and attach here to see what the updated output looks like?

**Comment by ashmeenkaur** [comment]
> @ashmeenkaur Is it possible to take a screenshot and attach here to see what the updated output looks like?

Added in the PR description.

**Review by sethiay — APPROVED** [review]
Relying on Nitin's review.

**Pull request merged**